great_expectations/data_context/store/expectations_store.py

```diff
@@ -30,6 +31,31 @@
     _TExpectation = TypeVar("_TExpectation", bound=Expectation)
 
 
+class ExpectationConfigurationDTO(pydantic.BaseModel):
+    class Config:
+        extra = pydantic.Extra.ignore
+
+    id: str
+    expectation_type: str
+    rendered_content: Optional[List[dict]]
```

Comment:
Isn't this just an optional dict? Where does the list come in?

Reply:
it actually [is a list of dicts](https://github.com/great-expectations/great_expectations/blob/4ecacb9c2eeba6d2ee16978da1f97ec4161426ae/great_expectations/expectations/expectation.py#L293)!